includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -318,4 +321,19 @@ private function getCommunityCentralLink() {
 			'href' => $this->getHref( 'community-central' ),
 		];
 	}
+
+	private function getSitenameData() {
+		$wgSitenameForWikiId = WikiFactory::getVarValueByName( 'wgSitename', $this->wikiId );
+
+		if ( $wgSitenameForWikiId ) {
+			$sitename = $wgSitenameForWikiId;
+		} else {
+			$sitename = $this->wg->Sitename;
+		}
+
+		return [
+			'type' => 'text',
+			'value' => $sitename,
```

Comment:
code above `return` could be replaced if you provide 4th param to `getVarValueByName` which is default.
So it could be `'value' => WikiFactory::getVarValueByName( 'wgSitename', $this->wikiId, true, $this->wg->Sitename );`